optuna/multi_objective/_hypervolume/base.py

```diff
@@ -4,11 +4,46 @@
 
 
 class BaseHypervolume(object, metaclass=abc.ABCMeta):
-    """Base class for hypervolume calculators"""
+    """Base class for hypervolume calculators
+
+        .. note::
+            In Optuna, this class is used for computing the hypervolume of objective points.
+            In other words, the coordinate of each point represents the one of the multi-objective
+            function's values.
+
+        .. note::
+            We assume that the each objective function to be minimized. If you use some objectives
+            to be maximized, please transform inputs to this class's `compute` method for the
+            minimization problem. For example,
+
+            .. code::
+                def objective(trial):
+                    ...
+                    return accuracy, elapsed_time
+
+                class YourSampler(BaseSampler):
+                    ...
+                    def sample_relative(study, trial, search_space):
+                        ...
+                        trials = study.get_trials()
+                        solution_sets = np.ndarray([t.values for t in trials])
+
+                        # Transform the objective to be maximized into that for minimization.
+                        solution_sets = np.ndarray([[-s[0], s[1]] for s in solution_sets])
+
+                        reference_point = 2 * np.max(solution_set, axis=0)
+                        hypervolume = WFG().compute(solution_sets, reference_point)
+                        ...
```

Comment:
This is just a suggestion but creating `YourSampler` to explain the usage of the hypervolume calculator is a bit complicated.

How about just calculating the hypervolume of the obtained Pareto solutions?  The code structure is relatively simpler and it may be more intuitive since the hypervolume is a kind of metric to show the performance of multi-objective optimization.

The following is just an example:
```suggestion
                study = optuna.multi_objective.create_study(["maximize", "minimize"])
                study.optimize(objective, n_trials=100)
                trials = study.get_pareto_front_trials()
                solution_sets = np.ndarray([t.values for t in trials])

                # Transform the objective to be maximized into that for minimization.
                solution_sets = np.ndarray([[-s[0], s[1]] for s in solution_sets])

                reference_point = 2 * np.max(solution_set, axis=0)
                hypervolume = WFG().compute(solution_sets, reference_point)
                print("Hypervolume of the Pareto solutions is {}".format(hypervolume))
```